<?xml version="1.0"?>

<launch>

    <arg name="veh" default="locobot" />
    <arg name="cmd" default="locobot/cmd_vel" />

    <include file="$(find subt_rl)/launch/pcToLaser_240_xbot.launch">
    </include>

    <group ns="$(arg veh)">

        <node pkg="subt_rl" type="goal_nav_tf_locobot.py" name="goal_nav_rl" output="screen">
            <remap from="goal_in" to="destination" />
            <remap from="laser_in" to="locobot/RL/scan" /> 
            <remap from="cmd_out" to="$(arg cmd)" />
        </node>

    </group>

</launch>
